ansible/roles/go-cli/tasks/deploy.yml

```diff
@@ -6,15 +6,39 @@
     path: "{{ go_cli_conf_dir }}"
     state: directory
 
+- name: "Ensure nginx directory for go-cli exists"
+  file:
+    path: "{{ go_cli_nginx_dir  }}"
+    state: directory
+
 - name: "Pull the {{ docker_image_tag }} image of go-cli"
   shell: "docker pull {{ docker_registry }}whisk/go-cli:{{ docker_image_tag }}"
   when: docker_registry != ""
 
-# TODO: Publish compressed binaries using nginx
```

Comment:
We want to have one Ansible command that copies the entire directories structure containing the compressed binaries to the nginx directory. Currently there are binary files inside the directories where the compressed binaries live. For example, we hope to run this command in Ansible: shell: "docker cp go-cli:/src/github.com/go-whisk-cli/bin '{{ nginx_conf_dir }}'". This will copy the directory structure, and just the compressed files to the nginx folder. We will have to restructure the directory content using the Dockerfile when Gradle performs the build, which will belong to a separate issue.